src/webpack.js

```diff
@@ -29,6 +29,21 @@ const checkFileExistAndPromptError = path => {
     return false;
   }
 };
+
+/**
+ * @desc Make sure directory existence
+ * @param {string} filePath
+ * @return {bool}
+ */
+const ensureDirectoryExistence = filePath => {
```

Comment:
you are not returning a bool. Since the tests are passing you didn't add a test case for this function I guess.

Reply:
also rename the function please